memconverter/converter.go

```diff
@@ -0,0 +1,87 @@
+// Package memconverter holds the conversion values to base unit (byte) for all
+// known units and makes the conversion to any unit.
+package memconverter
+
+var kb float64 = 1000
+var mb float64 = 1000000
+var gb float64 = 1000000000
+var tb float64 = 1000000000000
+var pb float64 = 1000000000000000
+
+var kib float64 = 1024
+var mib float64 = 1048576
+var gib float64 = 1073741824
+var tib float64 = 1099511627776
+var pib float64 = 1125899906842624
+
+func fromGB(num float64, unit string) (rez float64, success bool) {
```

Comment:
Please return an `error` instead of a `bool`.  

Useful:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings